src/worldfactory.cpp

```diff
@@ -710,14 +710,30 @@ void worldfactory::remove_world( const std::string &worldname )
 void worldfactory::load_last_world_info()
 {
     cata_path lastworld_path = PATH_INFO::lastworld();
-    if( !file_exist( lastworld_path ) ) {
-        return;
-    }
+    std::string lwmissing =
+        "lastworld.json or one of its values is empty.  This could be due to data corruption or an unknown error and may be an indicator that your previously played world is damaged.";
```

Comment:
```suggestion
        translate_marker( "lastworld.json or one of its values is empty.  This could be due to data corruption or an unknown error and may be an indicator that your previously played world is damaged." );
```